### PR TITLE
Adding node version information to the feed

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: cli-feed-v3.json [hunk 1]
```diff
@@ -127,6 +127,7 @@
     "2.2.2": {
       "Microsoft.NET.Sdk.Functions": "1.0.14",
       "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.31/Azure.Functions.Cli.win-x86.2.0.1-beta.31.zip",
+      "nodeVersion" : "8.11.1",
```
Comment (ahmelsayed) [inline cli-feed-v3.json]: actually I think this should be 10, right @mhoeger ?

Comment (mhoeger) [reply cli-feed-v3.json]: 8.11.1 is what the portal defaults to, and what we'd like :) after some discussion, we're going forward with supporting v8 and v10. The rule of thumb being we always support latest Active LTS (v8) and will try to support Current (v10), preference going to latest Active LTS. ([here's](https://github.com/nodejs/Release) the release schedule).

**Comment by jeffhollan** [comment]
@fiveisprime as FYI

**Comment by fiveisprime** [comment]
Nice timing, we're working on this right now. :)

**Comment by soninaren** [comment]
This is in prod now

**Pull request merged**